Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,408 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling;
+model Plant
+  "District cooling plant model"
+  extends Buildings.Experimental.DHC.CentralPlants.BaseClasses.PartialPlant(
```

Comment:
This model exposes the parameter `fue`. Is there any need for it? If not, set it to `final` so it does not show up in the parameter dialog.

Reply:
@ChengnanShi-Work
Could you comment on the use of `fue` in the `PartialPlant` class? If it's not currently used and is just a placeholder, I can hide it.

Reply:
Hi Baptiste, `fue` is not currently being used. We use electric chiller in our model and I believe we can hide it. Thank you!